src/python/WMCore/WMRuntime/Watchdog.py

```diff
@@ -78,6 +81,46 @@ def setupMonitors(self, task, wmbsJob):
                 #This should be a config section
                 monitorArgs = getattr(task.data.watchdog, monitor)
                 args = monitorArgs.dictionary_()
+                # Scale resources according to the HTCondor runtime environment.
+                origCores = 1
+                for stepName in task.listAllStepNames():
+                    sh = task.getStepHelper(stepName)
+                    origCores = max(origCores, sh.getNumberOfCores())
+                resources = {'cores': origCores}
+                origMaxRSS = args.get('maxRSS')
+                if origMaxRSS:
+                    origMaxRSS = int(origMaxRSS/1000.) # HTCondor expects MB; we get KB.
```

Comment:
To avoid future confusion, how about we adopt powers-of-two for the `origMaxRSS`, as we do for all the rest of the WMCore code (I think). Same comment valid for ~30 lines below.